Umbba-iOS/Umbba-iOS/Global/Resources/Lottie/ViewController/LottieViewController.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  LottieViewController.swift
+//  Umbba-iOS
+//
+//  Created by 고아라 on 2023/07/13.
+//
+
+import UIKit
+
+import SnapKit
+import Lottie
+
+final class LottieViewController: UIViewController {
+    
+    // MARK: - UI Components
+    
+    private let lottieView = LottieView()
+    
+    // MARK: - Life Cycles
+
+    override func loadView() {
+        super.loadView()
+
+        view = lottieView
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        playAnimation()
+    }
+}
+
+// MARK: - Extensions
+
+extension LottieViewController {
+    func playAnimation() {
+        lottieView.lottieSplashView.play { (finished) in
```

Comment:
```suggestion
        lottieView.lottieSplashView.play { _ in
```
- finished라는 이름의 인자를 사용하는 곳이 없기 때문에 생략해줘도 괜찮지 않을까 싶어요!